src/main/java/org/zalando/nakadi/exceptions/IllegalScopeException.java

```diff
@@ -0,0 +1,18 @@
+package org.zalando.nakadi.exceptions;
+
+import java.util.Set;
+
+public class IllegalScopeException extends RuntimeException {
```

Comment:
@v1ctor @adyach @v-stepanov @antban I thought that the team had chosen to use checked exceptions https://docs.google.com/spreadsheets/d/1p2GTg-fgrNGQBkhdM1yzKwvWOXvFL4FyiOoGtQJqTxI/edit#gid=0

But I personally liked this approach in this case, unchecked exceptions fits nicely 👍